app/controllers/api_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ApiController < ApplicationController
+  def service_interaction_for_local_authority
+    @authority = LocalAuthority.find_by_slug!(params[:authority_slug])
+    @service = Service.find_by(lgsl_code: params[:lgsl])
+    interactions = @service.interactions
+    @link = links.detect { |l| interactions.first.service_interactions.first.id == l.service_interaction.id }
+
+    render json: local_interaction_response
+  end
+
+private
+
+  def local_interaction_response
+    {
+      "local_authority"=>{
+        "name"=>@authority.name,
```

Comment:
What would we use the homepage URL for? We were thinking that we only need the homepage in case no valid LGIL could be found, in which case we would return the homepage url as the local_interaction url.